pkg/cloudprovider/huaweicloud/loadbalancer.go

```diff
@@ -19,15 +19,13 @@ package huaweicloud
 import (
 	"context"
 	"fmt"
-	"strconv"
-	"strings"
-
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/api/core/v1"
 	cloudprovider "k8s.io/cloud-provider"
 	"k8s.io/klog/v2"
+	"strconv"
```

Comment:
It's not a good import style, usually we move all imports in a single declaration, group stdlib imports, and group imports.
IDEA Setting: Settings -> Editor -> Code Style -> Go

Reply:
It has been fixed.